mapproxy/request/wms/__init__.py

```diff
@@ -185,11 +185,31 @@ class WMSMapRequest(WMSRequest):
     #pylint: disable-msg=E1102
     xml_exception_handler = None
     prevent_image_exception = False
+    dimension_params = ['time', 'elevation']
```

Comment:
What the reason behind adding these parameters?

Reply:
The reason is that only `time` and `elevation` can be used without a prefix - any other needs to be prefixed with `dim_` according to the spec:
```
A request parameter name is constructed by concatenating the prefix 'dim_' with the sample dimension Name (the value of the name attribute of the corresponding <Dimension> and <Extent> elements in the Capabilities XML).  The resulting "dim_name" is case-insensitive. The use of the 'dim_' prefix is to avoid clashes between server-defined dimension names and current or future OGC Web Service specifications.  (Time and Elevation, being predefined, do not use the prefix.) 
```
https://cdn.earthdata.nasa.gov/conduit/upload/514/01-068r3_web_map_service_implementation_specification.pdf